protocol/app/process/market_prices.go

```diff
@@ -63,9 +63,9 @@ func (umpt *UpdateMarketPricesTx) Validate() error {
 		return getValidateBasicError(umpt.msg, err)
 	}
 
-	if err := umpt.pricesKeeper.PerformStatefulPriceUpdateValidation(umpt.ctx, umpt.msg, true); err != nil {
-		return err
-	}
+	// if err := umpt.pricesKeeper.PerformStatefulPriceUpdateValidation(umpt.ctx, umpt.msg, true); err != nil {
+	// 	return err
+	// }
 
```

Comment:
This is temporary (just to make things work), we can remove this once we are aggregating prices that are proposed via `GetValidMarketPriceUpdates` instead of directly from sidecar